extensions/oidc-common/runtime/src/main/java/io/quarkus/oidc/common/runtime/OidcCommonConfig.java

```diff
@@ -13,84 +13,77 @@
 @ConfigGroup
 public class OidcCommonConfig {
     /**
-     * The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
-     * OIDC discovery endpoint will be called by default by appending a '.well-known/openid-configuration' path to this URL.
-     * Note if you work with Keycloak OIDC server, make sure the base URL is in the following format:
-     * `https://host:port/realms/{realm}` where `{realm}` has to be replaced by the name of the Keycloak realm.
+     * Optional. The base URL of the OpenID Connect (OIDC) server, for example, `https://host:port/auth`.
+     * The OIDC discovery endpoint is called by default by appending a `.well-known/openid-configuration` path to this URL.
+     * For Keycloak, use `https://host:port/realms/{realm}`, replacing `{realm}` with the Keycloak realm name.
      */
     @ConfigItem
     public Optional<String> authServerUrl = Optional.empty();
 
     /**
-     * Enables OIDC discovery.
-     * If the discovery is disabled then the OIDC endpoint URLs must be configured individually.
+     * Optional. Discovery of the OIDC endpoints.
+     * If not enabled, you must configure the OIDC endpoint URLs individually.
      */
     @ConfigItem(defaultValueDocumentation = "true")
     public Optional<Boolean> discoveryEnabled = Optional.empty();
 
     /**
-     * Relative path or absolute URL of the OIDC token endpoint which issues access and refresh tokens.
+     * Optional. The OIDC token endpoint that issues access and refresh tokens;
+     * specified as a relative path or absolute URL.
```

Comment:
Here and other similar properties it is worth adding something like: "Set if {@link #discoveryEnabled} is `false` or a discovered token endpoint path has to be customized"

Reply:
Thanks. I've added the line here. I'm unclear which other properties are similar and should receive this addition.
```suggestion
     * specified as a relative path or absolute URL.
     * Set if {@link #discoveryEnabled} is `false` or a discovered token endpoint path must be customized.
```

Reply:
@sberyozkin: I didn't close this because I couldn't find additional properties based on your suggestions.
I think you can mark this conversation as resolved if you're satisfied with the current state of the content.

Reply:
I believe this conversation is resolved. Please reopen it if needed.